GDM.HW4.OOP.Classes.Task3/Program.cs

```diff
@@ -0,0 +1,371 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GDM.HW4.OOP.Classes.Task3
+{
+    class Program
+    {
+        private static Shop _shop;
+        private static MobileStore[] _mobStore;
+        private static Phone[][] _phone;
+        static void Main(string[] args)
+        {
+            SetShopName();
+            SetNumberOfStores();
+            _mobStore = new MobileStore[_shop.NumberOfStores];
+            _phone = new Phone[_shop.NumberOfStores][];
+            ConsoleOptionMenu();
+
+            Console.ReadLine();
+        }
+        public static void ConsoleOptionMenu ()
```

Comment:
You named method like "Print in console menu option" but you method is doing everything for your shop. Please create new name and rewrite completely method and logic. And please always what is a name method and what actually method is doing inside!!!